### ISSUE TITLE
Handle crossref

### ISSUE THREAD

**Issue by JensNevens** [jwcarr/bibfish#7]
Several bib entries in my master bibtex file make use of 'crossref'. However, when using bibfish to extract the bib entries from a manuscript, the entries that are needed via crossref are not extracted. Would this be possible?

**Comment by jwcarr** [comment]
I'm not sure, can you give an example of a crossref bibtex entry? This might be an issue related to [bibtexparser](https://github.com/sciunto-org/python-bibtexparser), on which bibfish relies to parse the bibtex database.

**Comment by JensNevens** [comment]
Consider the following entries from the master bibtex file:

```
@inproceedings{vaneecke2017metalayer,
 author = {{Van Eecke}, Paul and Katrien Beuls},
 crossref = {aaai2017ss},
 pages = {258--265},
 title = {Meta-layer problem solving for computational construction grammar}
}

@proceedings{aaai2017ss,
 booktitle={The 2017 {AAAI Spring} Symposium Series},
 year={2017},
 publisher = {AAAI Press},
 venue={Stanford, CA, USA}
}
```

When I cite `vaneecke2017metalayer`, the bibtex entry for the proceedings `aaai2017ss` should also be copied by bibfish to the separate bibtex file. Without it, the reference of `vaneecke2017metalayer` is incomplete. Also, bibtex entries for proceedings, books, etc. should always be placed at the bottom of the bib file, in case multiple bibtex entries of papers use the same cross reference.

**Comment by jwcarr** [comment]
I see. Yes, this would be worth fixing, and probably wouldn't be too difficult. After extracting the cited entries from the master file, you'd check for any entries that contain `crossref` and then do a second extraction to pull out the crossref entries, and then merge the two sets together.

Is this something you'd be interested in working on? If not, I could take a look at it some time, but not sure when.

**Comment by jwcarr** [comment]
Closing this because support for crossrefs has now been implemented in version 0.3. But let me know if it's not working.
